README.md

```diff
@@ -25,17 +25,65 @@ const tokenPairsResponse = {
 };
 const validatorResult: ValidatorResult = validator.validate(tokenPairsResponse, relayerApiTokenPairsResponseSchema);
 ```
+### Pagination
```

Comment:
we might want to use cursors (https://docs.gdax.com/#pagination) instead because it is more suited for real-time data